### DISCUSSION TITLE
Plugin "Thefuck" stopped being updated and doesn't function. Should a plugin for a dead repo still be included in ohmyzsh??

### DISCUSSION THREAD

**Discussion by Arkenez** [ohmyzsh/ohmyzsh#12817] [Help]
Hello, First time I posted anything on GitHub. so please excuse any mistakes I may have made. I ran into multiple problems installing the "thefuck" plugin, and I couldn't get it to work because of a deprecated library called imp, which was removed in Python 3.10. 

I wanted to ask if a plugin should even be included anymore in the master repository if the original repo has stopped working.

Pull request for "thefuck" showing incompatibility with Python 3.12
https://github.com/nvbn/thefuck/pull/1475

GitHub repository:
https://github.com/nvbn/thefuck

Plugin Folder:
https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/thefuck

**Comment by roeniss** [comment]
_Whoa_. Last commit is [2024-01](https://github.com/nvbn/thefuck/commits/master/). Seems quite dead.
